sdk/python/sawtooth_sdk/consensus/zmq_driver.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import concurrent
+import logging
+from queue import Queue
+from threading import Thread
+
+from sawtooth_sdk.consensus.driver import Driver
+from sawtooth_sdk.consensus.zmq_service import ZmqService
+from sawtooth_sdk.consensus import exceptions
+from sawtooth_sdk.messaging.stream import Stream
+from sawtooth_sdk.protobuf import consensus_pb2
+from sawtooth_sdk.protobuf.validator_pb2 import Message
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ZmqDriver(Driver):
+    # pylint: disable=super-init-not-called
```

Comment:
I would still call the super init - event though it is an ABC with no implementation, there's no reason to avoid it.  Either that, or it doesn't seem like you need this disable comment, since you don't use it anywhere else.

Reply:
Took out the disable and added the super init call